galery.css

```diff
@@ -0,0 +1,3 @@
+.galery {
+  height: 10vw;
```

Comment:
will be invisible on mobiles ;)
by the way, we should always stick height to height and width to width (there is no link between them)
so when we want to set height we use `vh` and when width - `vw`